lua/bamboo/highlights.lua

```diff
@@ -64,6 +64,10 @@ local colors = {
 
 hl.common = {
   Normal = { fg = c.fg, bg = cfg.transparent and c.none or c.bg0 },
+  NormalNC = not cfg.transparent and cfg.dim_inactive and {
```

Comment:
Hmm, this would imply that you can't do both transparent=true, dim_inactive=true and honor both.